spec/support/helpers/dependency_helper.rb

```diff
@@ -115,6 +119,14 @@ def que_present?
     dependency_present? "que"
   end
 
+  def hanami_present?
+    dependency_present? "hanami"
+  end
+
+  def hanami2_present?
+    ruby_3_0_or_newer? && hanami_present? && Gem.loaded_specs["hanami"].version >= Gem::Version.new("2.0")
```

Comment:
Do we need/want to check for Ruby >= 3 explicitly? Hanami 2.0 is already only compatible with Ruby 3.0.

Reply:
This is to avoid earlier versions launching the suite. But maybe it is too defensive.